bower.json

```diff
@@ -32,7 +32,8 @@
     "iron-media-query": "polymerelements/iron-media-query#^1.0.8",
     "paper-ripple": "polymerelements/paper-ripple#^1.0.5",
     "iron-a11y-keys-behavior": "polymerelements/iron-a11y-keys-behavior#^1.1.1",
-    "iron-a11y-announcer": "PolymerElements/iron-a11y-announcer#^1.0.5"
+    "iron-a11y-announcer": "PolymerElements/iron-a11y-announcer#^1.0.5",
+    "iron-iconset-svg": "^1.1.0"
```

Comment:
please be consistent and add full package name before version: `PolymerElements/iron-iconset-svg`